test/verify/check-services

```diff
@@ -481,5 +481,66 @@ WantedBy=default.target
 
         self.allow_restart_journal_messages()
 
+    def testConditions(self):
+        m = self.machine
+        b = self.browser
+        m.write("/etc/systemd/system/condtest.service",
+"""
+[Unit]
+Description=Test Service
+ConditionDirectoryNotEmpty=/var/tmp/empty
+
+[Service]
+ExecStart=/bin/sh -c "while true; do sleep 5; done"
+
+[Install]
+WantedBy=default.target
+""")
+
+        m.execute("mkdir -p /var/tmp/empty")
+        m.execute("rm -rf /var/tmp/empty/*")
+
+        # After writing files out tell systemd about them
+        m.execute("systemctl daemon-reload")
+        self.machine.execute("systemctl start default.target")
+
+        self.login_and_go("/system/services")
+
+        def svc_sel(service):
+            return 'tr[data-goto-unit="%s"]' % service
+
+        def wait_service_state(service, state):
+            b.wait_present(svc_sel(service));
+            b.wait_in_text(svc_sel(service), state)
+
+        def wait_service_in_panel(service, title):
+            b.wait_in_text(".panel:contains('%s') .panel-heading" % service, title)
+
+        unit_action_btn = '#service-valid .list-group-item:nth-child(1) .btn-group'
+        file_action_btn = '#service-valid .list-group-item:nth-child(2) .btn-group'
+
+        # Selects Services tab
+        b.click('#services-filter :nth-child(2)')
+        b.wait_present(svc_sel('condtest.service'))
+        b.click(svc_sel("condtest.service"))
+        b.wait_visible('#service-valid')
+        b.wait_in_text('#service-valid', "inactive")
+        b.wait_action_btn(unit_action_btn, "Start")
+        b.click_action_btn(unit_action_btn)
+
+        b.wait_in_text('#service-unit-tmpl', "Condition failed")
```

Comment:
I don't think you want to check the template for the text. You need check where the template gets rendered in the dom.